### PR TITLE
Enter in some Curie bits

### PR DESCRIPTION
**Pull request by MoochMcGee**

### CODE REVIEW

File: hwtest/pgraph_scan.cc [hunk 1]
```diff
@@ -492,7 +499,7 @@ class ScanCelsiusTest : public ScanTest {
 }
 
 bool PGraphScanTests::supported() {
-	return chipset.card_type < 0x40;
+	return (chipset.card_type < 0x40);
```
Comment (koriakin) [inline hwtest/pgraph_scan.cc]: Any reason for that?

File: hwtest/pgraph_scan.cc [hunk 2]
```diff
@@ -107,7 +107,14 @@ class ScanDebugTest : public ScanTest {
 			bitscan(0x4000a0, 0xffffffff, 0);
 			bitscan(0x4000a4, 0xf, 0);
 			bitscan(0x4000c0, 0x1e, 0);
+		} else {
+			bitscan(0x400080, 0x00000107, 0);
+			bitscan(0x400084, 0x7010c7c1, 0);
+			bitscan(0x40008c, 0xe1fad155, 0);
+			bitscan(0x400090, 0x33ffffff, 0);
+			bitscan(0x4000c0, 0x00000006, 0);
```
Comment (koriakin) [inline hwtest/pgraph_scan.cc]: Please write which NV4x you tested this on - I'm going to guess it's wildly different between chipsets.

File: hwtest/pgraph_scan.cc (hunk 1 above)
Comment (MoochMcGee) [inline hwtest/pgraph_scan.cc]: Sorry, I accidentally left those in because i was trying to test JUST nv4e.

File: hwtest/pgraph_scan.cc (hunk 2 above)
Comment (MoochMcGee) [inline hwtest/pgraph_scan.cc]: Fixed.

**Pull request merged**